core/tests/unit/api_core/gapic/test_method.py

```diff
@@ -156,7 +170,12 @@ def test_wrap_method_with_overriding_retry_deadline(unusued_sleep):
 
     assert result == 42
     timeout_args = [call[1]['timeout'] for call in method.call_args_list]
-    assert timeout_args == [5, 10, 20, 29]
+    assert timeout_args == [5.0, 10.0, 20.0, 26.0, 25.0]
+    assert utcnow.call_count == (
+        1 +  # First to set the deadline.
+        5 +  # One for each min(timeout, maximum, (DEADLINE - NOW).seconds)
+        5
```

Comment:
@jonparrott These come somewhere from the `retry` module, but I'm not sure where.

11 calls seems to be a lot